site/documentation/content/api/tenant/index.md

```diff
@@ -316,6 +317,14 @@ The table below contains the properties that are used to configure a tenant's re
 | *mode*                   | *yes*     | *string*      | `-`           | The mode of the resource usage calculation. The default implementation supports two modes namely `days` and `monthly`. |
 | *no-of-days*             | *no*      | *number*      | `-`           | When the mode is set as `days`, then this value represents the length of an accounting period , i.e. the number of days over which the resource usage is to be limited. MUST be a positive integer.|
 
+### Extension properties Format
+
+The table below contains the extension properties that are used for addition configuration:
+
+| Name                          | Mandatory | JSON Type | Default Value | Description |
+|:------------------------------|:---------:|:----------|:--------------| :---------- |
+| *invalidate-cache-on-update*  |   *no*    | *boolean* | `false`       | Purge all cache entities from the ProtonBasedTenantClient on update operations at Tenant collection. |
```

Comment:
How about a description that does not involve any underlying implementation details? May be something likes this: _When invalidate-cache-on-update is set to true, the clients must purge the cached tenant entity for update operations._

Reply:
I tried to followed the style of the other boolean properties and to give a bit more meaning. What about:
_If set to `true`, the clients must purge the cached tenant entity when they receive change notification for update operation._

Reply:
@n-deliyski sounds good.

Reply:
done